Gemfile.lock

```diff
@@ -35,6 +35,8 @@ GEM
     docile (1.1.5)
     domain_name (0.5.25)
       unf (>= 0.0.5, < 1.0.0)
+    factory_girl (4.5.0)
```

Comment:
Yes, but this gem is a gem that is to be used as a client program... so nothing should depend on evaluator-slave.

Gemfile.lock should not be in the repo if it's a library and other things include it, because then the dependencies might not resolve.